custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -403,6 +400,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
     vm.setAvailableAndSelectedMonthForAWWPerformanceReport = function () {
         var today = new Date();
         if (vm.selectedYear === today.getFullYear()) {
+            vm.setMonthToPreviousIfBeforeThe15th(today);
```

Comment:
This name is pretty confusing given its current function

Reply:
changed